src/CoreWCF.Primitives/src/CoreWCF/EnableCoreWCFOperationInvokerGeneratorAttribute.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CoreWCF
+{
+    [AttributeUsage(AttributeTargets.Assembly)]
+    public sealed class EnableCoreWCFOperationInvokerGeneratorAttribute : Attribute
+    {
+        public string Value { get; }
+        public EnableCoreWCFOperationInvokerGeneratorAttribute(string value)
```

Comment:
Is the type of Value a string to enable more nuanced options than true and false in the future?

Reply:
updated to use a boolean value